tslint.json

```diff
@@ -30,7 +30,8 @@
     "object-shorthand-properties-first": false,
     "prefer-array-literal": [true, { "allow-type-parameters": true }],
     "array-type": false,
-    "member-ordering": false
+    "member-ordering": false,
+    "no-this-assignment": false
```

Comment:
This rule should not be disabled. It complains that in some classes (all day panel, views) an appointment placeholder is assigned to a variable - we should fix that

Reply:
.. or disable the rule in those places

Reply:
This rule can be configured in the following way: 
"no-this-assignment": [true, {"allow-destructuring": true}]